Dockerfile

```diff
@@ -4,4 +4,4 @@ COPY tests/tests.test .
 RUN mv tests.test /bin
 RUN apt-get update -y && apt-get install -y curl openssh-client git
 RUN curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | bash && mv ./deis /bin/deis
-CMD /bin/tests.test
+CMD ["/bin/tests.test", "-v", "-timeout=60m", "-ginkgo.v", "-ginkgo.slowSpecThreshold=120"]
```

Comment:
I guess by baking the args into the default Dockerfile `CMD`, I was hoping this would Just Work&trade; for the deis-tests chart. But it appears I will have to make similar changes [over there](https://github.com/deis/charts/blob/master/deis-tests/manifests/deis-tests-pod.yaml#L13-L14).